packages/contracts-bedrock/scripts/outputs/DeleteOutput.s.sol

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { console } from "forge-std/console.sol";
+import { Script } from "forge-std/Script.sol";
+import { IMulticall3 } from "forge-std/interfaces/IMulticall3.sol";
+
+import { Types } from "../contracts/libraries/Types.sol";
+import { FeeVault } from "../contracts/universal/FeeVault.sol";
+import { Predeploys } from "../contracts/libraries/Predeploys.sol";
+
+interface IL2OutputOracle {
+    function deleteL2Outputs(uint256 _outputIndex) external;
+    function CHALLENGER() external view returns (address);
+    function getL2Output(uint256 _l2OutputIndex) external view returns (Types.OutputProposal memory);
+}
+
+/**
+ * @title DeleteOutput
+ * @notice Deletes an output root from the L2OutputOracle.
+ * @notice Example usage is provided in the README documentation.
+ */
+contract DeleteOutput is Script {
```

Comment:
If this inherits from `SafeBuilder` then we don't need to copy/paste the `run` logic